test/metabase/setup_test.clj

```diff
@@ -97,3 +102,35 @@
           (is (true? (mi/can-write? sample-db)))
           (is (true? (mi/can-write? sample-db-table)))
           (is (true? (mi/can-write? sample-db-field))))))))
+
+(deftest encryption-test
+  (mt/test-drivers #{:h2 :mysql :postgres}
+    (testing "Database can start with no encryption"
+      (encryption-test/with-secret-key ""
```

Comment:
nit - looks like you can equivalently use `nil` here for the secret key, which might be slightly clearer in intent?

or better yet you could add a `with-no-secret-key` (or equivalent) wrapper macro to make it super clear. but not a huge deal either way.